tool/update_perfetto.sh

```diff
@@ -45,6 +45,9 @@ find ./ -name '*.js.map' -exec rm {} \;
 find ./ -name 'traceconv.wasm' -exec rm {} \;
 find ./ -name 'traceconv_bundle.js' -exec rm {} \;
```

Comment:
nit can you move these down to the deleting catapult files? these are for the legacy UI as well

Reply:
Done.